app/views/content/train-to-be-a-teacher/school-placements.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Teacher training school placements"
+heading: "Teacher training school placements"
+subcategory: Postgraduate teacher training
+description: |-
+  Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available. 
+navigation: 20.20
+navigation_title: Teacher training school placements
+navigation_description: Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available.
+quote:
+  jennifer:
+    text: "My mentor was highly knowledgeable and experienced, and had high expectations which helped me to flourish."
+    name: "Jennifer, trainee teacher"
+    classes: quote--indent
+  aaron:
+    text: "My first placement was an academy with an experienced teacher in a year 3 classroom. She took me under her wing and taught me all the basics on how to conduct myself in the classroom along with how to best motivate and nurture the children to promote positive behaviour responses and enhance the classroom culture."
+    name: "Aaron, trainee teacher"
+    classes: quote--indent
+  abbe:
+    text: "My mentor was the special educational needs co-ordinator. This meant I was able to learn more about this role and improve my understanding of the strategies and support available for children with special educational needs and disabilities (SEND)."
+    name: "Abbe, trainee teacher"
+    classes: quote--indent
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+keywords:
+    - placements
+    - school placements
+    - classroom placements
+    
+---
+
+School placements are an important and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:
+
+* get experience in the classroom
+* immerse yourself in your placement schools
+* get to know your pupils and see their progress
+* be part of a school community and learn from other teachers
+* start your career as a teacher
+
+$aaron$
+
+## How long are placements?
+
+School placements last for a minimum of 24 weeks. This time is spread across at least 2 schools. This is to give you a varied and thorough experience in the classroom. 
+
+For example, in your first placement you might teach key stage 4, and in your second placement you might teach key stage 3 in a school for pupils with [special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs).
+
+You'll need experience of teaching in 2 schools to be awarded [qualified teacher status](/train-to-be-a-teacher/what-is-qts).
+
+## Where will my placements take place?
+
+Where you complete your school placements will depend on whether you've chosen a primary or secondary teacher training course.
```

Comment:
This sounds slightly odd @emmaachesongray - it is not just dependent on primary vs secondary (although that is obviously a factor!). However, I appreciate this is what is in prod at the moment!